airbyte-integrations/connectors/source-sftp-bulk/metadata.yaml

```diff
@@ -22,8 +24,13 @@ data:
     oss:
       enabled: true
   releaseStage: alpha
+  releases:
+    breakingChanges:
+      1.0.0:
+        message: "This upgrade brings migration to File-Based CDK"
```

Comment:
This upgrade migrates the SFTP Bulk source to the Airbyte file-based CDK.  This is the first necessary step of transitioning a file connector from community to Airbyte maintained.

Reply:
updated